openapi.yml

```diff
@@ -418,23 +491,111 @@ components:
           "ipaCode": "c_e956",
           "legalRepresentative": {
             "familyName": "Sergio",
-            "firstName": "Mazzia",
+            "givenName": "Mazzia",
             "fiscalCode": "BCDFGH12A21Z123E",
             "phoneNumber": "3333333333"
           },
           "name": "Comune di Marina di Gioiosa Ionica",
-          "pecs": [
-            "indirizzo20@email.pec.it",
-            "indirizzo21@email.pec.it",
-          ],
+          "pecs": {
+            "1": "indirizzo20@email.pec.it",
+            "2": "indirizzo21@email.pec.it",
+          },
           "scope": "LOCAL",
-          "selectedPecIndex": 0,
+          "selectedPecLabel": "1",
```

Comment:
Question: do we need `selectedPecLabel` in the application logic? Can't we save the actual pec address so that even in case of ambiguity we don't pick the wrong address?